app/komponenter/feilside/Feilside.tsx

```diff
@@ -0,0 +1,21 @@
+import { GuidePanel } from '@navikt/ds-react';
+import { Dokument } from '~/root';
+import { LocaleType } from '~/typer/sanity/sanity';
+import Banner from '../banner/Banner';
+import HovedInnholdCss from '../hovedInnhold/hovedInnhold.module.css';
+import css from './feilside.module.css';
+
+const Feilside: React.FC = () => {
+  return (
+    <Dokument språk={LocaleType.nb}>
+      <Banner bannerTekst={'Endringsmelding for barnetrygd'} />
+      <main className={`${HovedInnholdCss.innholdKonteiner}`}>
+        <GuidePanel className={`${css.feilmeldingPanel}`}>
+          <p>En feil har oppstått! Vennligst prøv igjen.</p>
```

Comment:
Vi burde lage et kort på å lage semantiske navn for css.

Reply:
Enig, fikset her: https://trello.com/c/32K5PVCG/90-endre-til-sematiske-navn-for-css